README.md

```diff
@@ -597,10 +603,16 @@ The tasks are stored in a cron file.
 
 <details>
 <summary>How to add a new user to the system without providing him the ability to log-in into the system?</summary><br><b>
+
+  * adduser user_name --shell=/bin/false --no-create-home
+
 </b></details>
 
 <details>
 <summary>On a system which uses systemd, how would display the logs?</summary>
+
+  * journactl
```

Comment:
journalctl*

Reply:
Thanks @bregman-arie , Could you check it again?